catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -265,6 +265,24 @@ public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 			copy.look.setLookData(copy.getLookDataList().get(0));
 		}
 
+		copy.look.setPositionInUserInterfaceDimensionUnit(cloneMe.look.getXInUserInterfaceDimensionUnit(),
+				cloneMe.look.getYInUserInterfaceDimensionUnit());
+		copy.look.setTransparencyInUserInterfaceDimensionUnit(cloneMe.look.getTransparencyInUserInterfaceDimensionUnit());
+		copy.look.setColorInUserInterfaceDimensionUnit(cloneMe.look.getColorInUserInterfaceDimensionUnit());
+
+		int rotationMode = cloneMe.look.getRotationMode();
+		copy.look.setRotationMode(rotationMode);
+		if (rotationMode != Look.ROTATION_STYLE_LEFT_RIGHT_ONLY && copy.look.isFlipped()) {
+			copy.look.getLookData().getTextureRegion().flip(true, false);
+			copy.look.setFlipped(false);
+		}
+		boolean orientedLeft = copy.look.getDirectionInUserInterfaceDimensionUnit() < 0;
+		if (rotationMode == Look.ROTATION_STYLE_LEFT_RIGHT_ONLY && orientedLeft) {
+			copy.look.getLookData().getTextureRegion().flip(true, false);
+			copy.look.setFlipped(true);
+		}
+		copy.look.setDirectionInUserInterfaceDimensionUnit(cloneMe.look.getDirectionInUserInterfaceDimensionUnit());
+
```

Comment:
Both, cloning a look through [the IDE](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Sprite.java#L335) or [the clone brick](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Sprite.java#L362), will call [Sprite.cloneLooks(Sprite)](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Sprite.java#L490) which will call [Look.copyLookForSprite(Sprite)](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Look.java#L132) which already sets some of the look members. For consistency and also to fix [CAT-1384](https://jira.catrob.at/browse/CAT-1384), it would be awesome if you adapt the [Look.copyLookForSprite(Sprite)](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Look.java#L132) method instead.